Modules/_elementtree.c

```diff
@@ -3313,14 +3324,26 @@ _elementtree_XMLParser___init___impl(XMLParserObject *self, PyObject *html,
     self->target = target;
 
     self->handle_start = PyObject_GetAttrString(target, "start");
+    if (ignore_attribute_error(self->handle_start))
```

Comment:
New PEP 7 rules require braces even around simple `if` body.

Reply:
Done. (Sad in this case, given the relation between "active" code and error handling code.)